generator/ServiceClientGenerator/App.config

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.1"/>
```

Comment:
Don't check this file in. While not a bad idea to update our generator to newer versions of .NET Framework, we should do this a separate effort.